.gitignore

```diff
@@ -3,3 +3,4 @@ _build
 *.merlin
 _opam
 *.events
+.vscode
```

Comment:
Do you have a global gitignore file? Or could set one up? Given that `.vscode` is user specific rather project specific, it would be more hygienic to add it to your global gitignore. FWIW (for what it's worth), I myself have `.vscode/*` in my global gitignore as well.

Reply:
I wasn't doing this, thanks for this tip, it'll save me time in a lot of cloned repos!!